src/macpro-security-hub-sync.ts

```diff
@@ -333,13 +333,13 @@ export class SecurityHubJiraSync {
     let newIssueInfo;
     try {
       newIssueInfo = await this.jira.createNewIssue(newIssueData);
-      const feature_key = process.env.JIRA_LINK_ID ?? "";
-      if (feature_key) {
+      const issue_key = process.env.JIRA_LINK_ID ?? "";
```

Comment:
minor ask but can this be `issue_id` instead of `issue_key` to keep it consistent with the other names?